test/parallel/test-eval.js

```diff
@@ -1,13 +1,16 @@
 'use strict';
 const common = require('../common');
-var util = require('util');
-var assert = require('assert');
-var exec = require('child_process').exec;
+const util = require('util');
+const assert = require('assert');
+const exec = require('child_process').exec;
 
-var cmd = ['"' + process.execPath + '"', '-e', '"console.error(process.argv)"',
-           'foo', 'bar'].join(' ');
-var expected = util.format([process.execPath, 'foo', 'bar']) + '\n';
-exec(cmd, common.mustCall(function(err, stdout, stderr) {
+const cmd = [
+  `"${process.execPath}"`,
+  '-e',
+  '"console.error(process.argv)"',
+  'foo', 'bar'].join(' ');
```

Comment:
Nit: there is one element per line now, what about putting `'bar'` on a new line?